exe/server/Dockerfile

```diff
@@ -6,12 +6,12 @@ COPY extern extern
 COPY go.mod go.sum ./
 RUN go mod download
 COPY . . 
-RUN GOOS=linux go build -o main exe/server/main.go && \
-go run github.com/GeertJohan/go.rice/rice append --exec main -i ./build
+RUN GOOS=linux go build -o fil-tools exe/server/main.go && \
```

Comment:
how about `fil-toolsd` since in some cases you'll be running the server and cli (presumably called `fil-tools`) on the same computer?

Reply:
If this only applies to docker, maybe it doesn't matter, don't know.

Reply:
Yeah, actually mostly applies to docker since the binary isn't use elsewhere. 
Despite that, I like the convention. Will change.

Reply:
cool